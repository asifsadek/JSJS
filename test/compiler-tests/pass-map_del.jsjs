val a = { "foo" : "bar"};
print(del(a, "foo"));
print(del(a, "bar"));

val b = { 1 : [1, 2, 3] };
print(del(b, 1));
print(del(b, 2));

val c = { true : 2,
          false : 3 };
print(del(c, true));

val d = { 1 : { "foo" : 2}};
print(del(d, 1));
print(del(d, 2));
