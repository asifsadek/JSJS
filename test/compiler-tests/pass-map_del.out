Map {}
Map { "foo": "bar" }
Map {}
Map { "1": List [ 1, 2, 3 ] }
Map { "false": 3 }
Map {}
Map { "1": Map { "foo": 2 } }
